app/src/composables/useSiwe.ts

```diff
@@ -49,12 +49,9 @@ async function siwe() {
 
     router.push('/teams')
   } catch (error: any) {
+    log.error(parseError(error), 'Some random error message', new Error('Example error'))
```

Comment:
i don't get why you have 3 input for the error here. 
I think you do it just for demo.

Reply:
Yeah it was just a demo, I'll remove that